README.md

```diff
@@ -1 +1,103 @@
-# customerio-reactnative
\ No newline at end of file
+<p align="center">
+  <a href="https://customer.io">
+    <img src="https://user-images.githubusercontent.com/6409227/144680509-907ee093-d7ad-4a9c-b0a5-f640eeb060cd.png" height="60">
+  </a>
+  <p align="center">Power automated communication that people like to receive.</p>
+</p>
+
+![min swift version is 5.3](https://img.shields.io/badge/min%20Swift%20version-5.3-orange)
+![min ios version is 13](https://img.shields.io/badge/min%20iOS%20version-13-blue)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-4baaaa.svg)](code_of_conduct.md) 
+[![codecov](https://codecov.io/gh/customerio/customerio-reactnative/branch/main/graph/badge.svg?token=IZ9RP9XD1O)](https://codecov.io/gh/customerio/customerio-reactnative)
+
+# Customer.io React Native SDK
+
+This is the official Customer.io SDK for React Native.
+
+You'll find our [complete SDK documentation at https://customer.io/docs/sdk/react-native](https://customer.io/docs/sdk/react-native/). This readme only contains basic information to help you install and initialize the SDK.
+
+## Install and initialize the SDK
+
+1. Open your terminal and run `npm install customerio-reactnative`
+1. Add iOS dependencies to your project by going to the iOS subfolder and running `pod install`. 
+
+   Make sure your deployment target is set to at least 13.0. Before you perform this step, you may want to update your podfile to support [APNs and/or FCM push notifications](https://customer.io/docs/sdk/react-native/push/#install-the-push-package) and [rich push](https://customer.io/docs/sdk/react-native/rich-push/#rich-push) respectively.
+
+1. For Android, add the following lines to the project-level `android/build.gradle` file:  
+      ```groovy
+      buildscript {
+         repositories {
+            // Add this line if it isn't already in your build file:
+            google()  // Google's Maven repository
+         }
+
+         dependencies {
+            // Add this line:
+            classpath 'com.google.gms:google-services:4.3.13'  // Google Services plugin
```

Comment:
do we have a dependency on this version of the google services plugin?
if not maybe replace the `4.3.13` with `MAJOR.MINOR.PATCH` instead

Reply:
Thanks for pointing this out Jatin. I see we need to make this change here as well as in our docs, I'll do the changes for both.

Reply:
[Removed](https://github.com/customerio/customerio-reactnative/pull/85/commits/05468528fb63bf3ae61defd1a84ed24657a205db) the exact version